prj_shun/nCov_survey.py

```diff
@@ -0,0 +1,453 @@
+import openpyxl
+import numpy as np
+import pandas as pd
+import glob
+
+PATH = "./nCoV_survey_files"
+
+sheet_1_name = "患者・疑似症患者臨床症状調査（添付１）"
+sheet_4_name = "接触者リスト（添付3-2）"
+
+def main():
+    path2xlsx = PATH + "/*.xlsx"
+    paths = glob.glob(path2xlsx)
+    
+    df1 = pd.DataFrame()
+    df2 = pd.DataFrame()
+    
+    for path in paths:
+        df1 = createMedicalHistoryDF(path,df1 )
+        df2 = createContactPersonsDF(path,df2 )
```

Comment:
ファイルの読み込みには時間が掛かります。
なので、読み込みは最低限の回数で行いたいです。
wb, ws をここに入れたい。
```suggestion
    for path in paths:
        wb = ~.workbook(path) 
        df1 = createMedicalHistoryDF(wb,df1 )
        df2 = createContactPersonsDF(wb,df2 )
```
みたいな。